#!/usr/bin/env python
PACKAGE = "runner"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("max_width",   int_t,    0,
        "Max panorama width (pixels)", 2000, 100, 10000)
gen.add("auto_reset",  bool_t,   0,
        "Reset panorama on each new frame", False)
gen.add("reset_now",   bool_t,   0,
        "Immediately reset panorama", False)
gen.add("min_shift",   int_t,    0,
        "Ignore shifts smaller than this (px)", 5, 0, 100)
gen.add("max_shift",   int_t,    0,
        "Ignore shifts larger than this (px)", 500, 0, 2000)
exit(gen.generate(PACKAGE, "runner", "FoamStitch"))
